src/components/Icon/Icon.tsx

```diff
@@ -6,10 +6,9 @@ import { getNativeAttributes, htmlElementAttributes } from '../../utilities/attr
 import './Icon.scss';
 
 export const Icon: (props: IIconProps) => JSX.Element = (props: IIconProps) => {
-
-    let size = props.size;
-    let customIcon = props.iconName === '';
-    let iconClass = props.iconName;
+    // let size = props.size;
+    const customIcon = props.iconName === '';
+    // let iconClass = props.iconName;
```

Comment:
You can delete these commented lines?
It does seem that these changes will not affect current usage of Icon component.